### PR TITLE
input-item translation, ref #329

### PR DESCRIPTION
**Pull request by silentcloud**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/1297)

### CODE REVIEW

File: components/input-item/__tests__/__snapshots__/demo.test.web.js.snap [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports[`renders ./components/input-item/demo/basic.md correctly 1`] = `
     <div
       class="am-list-header"
     >
-      自定义获取光标
+      Customize to get the cursor
```
Comment (paranoidjk) [inline components/input-item/__tests__/__snapshots__/demo.test.web.js.snap]: Customize to focus

File: components/input-item/__tests__/__snapshots__/demo.test.web.js.snap [hunk 2]
```diff
@@ -147,7 +147,7 @@ exports[`renders ./components/input-item/demo/basic.md correctly 1`] = `
           class="am-input-control"
         >
           <input
-            placeholder="带清除按钮，输入会显示"
+            placeholder="displayed while typing"
```
Comment (paranoidjk) [inline components/input-item/__tests__/__snapshots__/demo.test.web.js.snap]: displayed clear icon while typing

File: components/input-item/index.en-US.md [hunk 3]
```diff
@@ -20,26 +18,26 @@ Support WEB, React-Native.
 
 Properties | Descrition | Type | Default
 -----------|------------|------|--------
-| type    | 银行卡`bankCard`,手机号`phone`（此时最大长度固定为11,`maxLength`设置无效）,密码`password`, 数字`number`（尽量唤起数字键盘）  | String |  `text`  |
-| value    | value 值(受控与否参考https://facebook.github.io/react/docs/forms.html)  | String |  无  |
-| defaultValue    | 设置初始默认值        | String |  -  |
+| type    | the type of `InputItem` which is one of `bankCard`, `phone` which the maxLength is 11 and setting will be ignored, `password`, `number`, `text`  | String |  `text`  |
```
Comment (paranoidjk) [inline components/input-item/index.en-US.md]: 这里容易误解，phone的特殊规则单独在括号里面写吧

File: components/input-item/index.en-US.md [hunk 4]
```diff
@@ -20,26 +18,26 @@ Support WEB, React-Native.
...
-| extra       | 右边注释   | string or node |  ''  |
-| onExtraClick      | extra 点击事件触发的回调函数 | (e: Object): void |  无  |
-| labelNumber  | 定宽枚举值：`num * @input-label-width: 34px`，可用`2-7`之间的数字，一般(不能保证全部)能对应显示出相应个数的中文文字(不考虑英文字符) | number | `5` |
-| autoFocus   | 页面初始化时Input自动获取光标,每个页面只有一个Input的autpFocus会生效。（不保证所有浏览器都生效） | bool | false  |
-| focused   | 页面运行过程中,Input获取光标,当Input获取光标（`focused`更新为true）后，需要在`onFocus`或者`onBlur`时再次将该属性设置为false。 | bool | false  |
-| updatePlaceholder (`web only`) | 当清除内容时，是否将清除前的内容替换到 placeholder 中 | bool |  false  |
-| prefixListCls (`web only`)    |   列表 className 前缀      | String |  `am-list`  |
-| name (`web only`)   | input 的 name        | String |  无  |
-
-> 更多属性请参考 react-native TextInput (http://facebook.github.io/react-native/docs/textinput.html)
```
Comment (paranoidjk) [inline components/input-item/index.en-US.md]: web 也支持 react-native TextInput 的所有 API 的意思？

Comment (silentcloud) [inline components/input-item/index.en-US.md]: 这个应该是针对 rn 的，我等会去备注一下

**Pull request merged**